sample/src/test/kotlin/io/github/bluegroundltd/sample/MyEnumTest.kt

```diff
@@ -0,0 +1,30 @@
+package io.github.bluegroundltd.sample
+
+import org.junit.Test
+
+class MyEnumTest {
+
+    @Test
+    fun `should map string value to enum value`() {
+        // Given
+        val firstEntryValue = MyEnum.FIRST_ENTRY.value
+
+        // When
+        val firstEntryEnumValue = MyEnumMapper.fromValue(value = firstEntryValue)
+
+        // Then
+        assert(firstEntryEnumValue == MyEnum.FIRST_ENTRY)
```

Comment:
💡 I think that we could use `assertEquals(MyEnum.FIRST_ENTRY, firstEntryEnumValue)` in these test cases